build_tools/benchmarks/common/common_arguments.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+import argparse
+
+
+def build_common_argument_parser():
```

Comment:
I'm assuming all this is just copied over. I really wish GitHub made it easier to review that. If there are any changes from how it was in the other file, can you flag them for review?

Reply:
Yes, it is just copied over.